google/compute_shared_operation.go

```diff
@@ -5,6 +5,25 @@ import (
 	"google.golang.org/api/compute/v1"
 )
 
+func computeSharedOperationWait(config *Config, op interface{}, project string, activity string) error {
```

Comment:
For sure - what I'd like to do is convert the `Operation` object to `v1` and call the `v1` waits, like in [#187](https://github.com/terraform-providers/terraform-provider-google/pull/187/files#diff-ef92ceb02a5a2c0c742317597b0bc870R104)

Otherwise, we can just copy `compute_operation.go` again.